operations/customers.md

```diff
@@ -170,6 +172,9 @@ Returns all customers filtered by identifiers, emails, names and other filters.
 | `ItalianDestinationCode` | string | optional | Value of Italian destination code. |
 | `ItalianFiscalCode` | string | optional | Value of Italian fiscal code. |
 | `CompanyId` | string | optional | Unique identifier of [Company](companies.md#company) the customer is associated with. |
+| `MergeTargetId` | string | optional | `Id` of the account (Customer/Company) where the customer is linked to. |
```

Comment:
Two small suggestions:
1. Grammar fix - better to say:
| `MergeTargetId` | string | optional | `Id` of the account (customer/company) to which this customer is linked. |

2. Can we include a link to 'account', does it exist as an entity in the API? Or is it some sort of co-product for Company and Customer? But would it always be Customer in this case, because the name "MergeTargetId" makes me think that this is used when merging two customer profiles? I don't know, but I just don't want it to be confusing for the user. Thanks.

Reply:
word fixed. The account doesn't exist, it is an entity in our systems - it is Customer / Company.. but maybe can be in the future something different. We can remove that word account for clarity, or keep it as it is with these examples..

Reply:
Thanks @pindurav. Just to be clear, are you saying that there is no such entity or object as account, it's just a word used to mean either Company or Customer? That's ok, but my next question then is how does the user know if `MergeTargetId` points to a Customer ID or a Company ID?